### ISSUE TITLE
Update to Stock Synthesis 3.30.18

### ISSUE THREAD

**Issue by k-doering-NOAA** [nmfs-fish-tools/SSMSE#87]
This should help us keep up to date with SS. will need to re-run built in models, as I believe the par file format has changed.

**Comment by nathanvaughan-NOAA** [comment]
I agree, keeping pace with the SS releases seems like the best option to
me.

**Comment by rwildermuth** [comment]
I don't think this would cause problems for us. I thought we had updated the SS files to work with SS3.30.17, so hopefully it would still work fine with the newer version. What's currently being used in the SSMSE main branch?
